modules/05_initialCap/on/preloop.gms

```diff
@@ -532,12 +532,24 @@ if (cm_startyear gt 2005,
   Execute_Loadpoint 'input_ref' pm_EN_demand_from_initialcap2 = pm_EN_demand_from_initialcap2;
   Execute_Loadpoint 'input_ref' pm_pedem_res = pm_pedem_res;
   Execute_Loadpoint 'input_ref' pm_dataeta = pm_dataeta;
-  Execute_Loadpoint 'input_ref' pm_data = pm_data;
   Execute_Loadpoint 'input_ref' pm_aux_capLowerLimit = pm_aux_capLowerLimit;
   Execute_Loadpoint 'input_ref' vm_deltaCap.l = vm_deltaCap.l;
   Execute_Loadpoint 'input_ref' vm_deltaCap.lo = vm_deltaCap.lo;
   Execute_Loadpoint 'input_ref' vm_deltaCap.up = vm_deltaCap.up;
 
+
+
+*** load pm_data from input_ref.gdx and overwrite values
+*** only for eta of chp technologies since they have been adapted in initialCap routine above
```

Comment:
I would pull "only for eta" back to the line before, otherwise it can be misunderstood